### PR TITLE
Homework1

### PR DESCRIPTION
**Pull request by jelenanemcic**

### CODE REVIEW

File: homework_1/zad1/fizzbuzz/fizzbuzz_test.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+			if !areSlicesEqual(actualOutput, tc.expectedOutput) {
+				t.Errorf(
+					"Actual and expected output is not equal - actual: %v, expected: %v",
+					actualOutput,
+					tc.expectedOutput)
+			}
+		}
+	}
+}
```
Comment (mbekavac-happening) [inline homework_1/zad1/fizzbuzz/fizzbuzz_test.go]: please add newline to the end of files

File: homework_1/zad1/fizzbuzz/fizzbuzz_test.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package fizzbuzz_test
+
+import (
+	"code-cadets-2021/homework_1/zad1/fizzbuzz"
+	"testing"
+)
+
+func areSlicesEqual(first []string, second []string) bool {
```
Comment (mbekavac-happening) [inline homework_1/zad1/fizzbuzz/fizzbuzz_test.go:9]: If you used goconvey (not sure if it was covered on Saturday, but I've seen some other people using it), you would have gotten this for free. Anyway, it was nicely written.

File: homework_1/zad1/fizzbuzz/testCases_test.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+type testCase struct {
+	inputStart   int
+	inputEnd     int
+
+	expectedOutput []string
+	expectingError bool
+}
+
+func getTestCases() []testCase {
```
Comment (mbekavac-happening) [inline homework_1/zad1/fizzbuzz/testCases_test.go:11]: what about negative numbers?

File: homework_1/zad1/main.go [hunk 4]
```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/pkg/errors"
```
Comment (mbekavac-happening) [inline homework_1/zad1/main.go]: Please set your IDE to sort the imports. Running `go fmt` should be sufficient.

File: homework_1/zad1/main.go [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+func main() {
+	var start, end int
+
+	flag.IntVar(&start, "start", 1, "Starting value for game Fizzbuzz")
+	flag.IntVar(&end, "end", 10, "Ending value for game Fizzbuzz")
+
+	flag.Parse()
+
+	result, err := fizzbuzz.PlayFizzBuzz(start, end)
```
Comment (mbekavac-happening) [inline homework_1/zad1/main.go]: `PlayFizzBuzz` is not too informative - it sounds like a function that does some [side effects](https://softwareengineering.stackexchange.com/questions/40297/what-is-a-side-effect), while it's actually a [pure function](https://en.wikipedia.org/wiki/Pure_function) that gets, calculates or generates FizzBuzz output. Please rename accordingly.

**Review by mbekavac-happening — CHANGES_REQUESTED** [review]

File: homework_1/zad1/fizzbuzz/fizzbuzz.go [hunk 6]
```diff
@@ -0,0 +1,29 @@
+package fizzbuzz
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+)
+
+func PlayFizzBuzz(start, end int) ([]string, error) {
```
Comment (mbekavac-happening) [inline homework_1/zad1/fizzbuzz/fizzbuzz.go]: additional note - please write docstrings for all public functions

File: homework_1/zad1/fizzbuzz/testCases_test.go (hunk 3 above)
Comment (jelenanemcic) [reply homework_1/zad1/fizzbuzz/testCases_test.go:11]: Thank you, I will add a test case for negative numbers.

**Review by mbekavac-happening — APPROVED** [review]
